learn/wavemaker-release-notes/v11-2-0.md

```diff
@@ -11,22 +11,25 @@ New features, such as One-click API Mocking, React Native improvements, and a fe
 
 ### Enable API Mocking in WaveMaker Studio
 
-API Mockingbird can mock APIs with real-time data to work concurrently on the front-end of an application, while the back-end team develops the API. Learn more about how to [enable API Mocking with a single-click](/learn/app-development/services/mock-services/mock-imported-apis).
+API Mockingbird can mock APIs with realistic data, allowing you to work concurrently on the front-end of an application, while the back-end team develops the API. Learn more about how to [enable API Mocking with a single-click](/learn/app-development/services/mock-services/mock-imported-apis).
 
 
 ### Easily Discover Services and Methods for API Composition
 
-You can compose a Java API with easy [discovery of dependencies](/learn/app-development/services/java-services/api-composer-toolkit#discover-dependencies) from the right panel for API Composition. Learn more about how to [Discover and Compose a Java API](/learn/app-development/services/java-services/api-composer-toolkit). 
+You can compose a Java API with easy [discovery of dependencies](/learn/app-development/services/java-services/api-composer-toolkit#discover-dependencies) from the right panel, and with a single-click you to generate code for **Import** statement and **@Autowire** annotation for the Service. Learn more about how to [Discover and Compose a Java API](/learn/app-development/services/java-services/api-composer-toolkit). 
 
+### TrueSPA
```

Comment:
**Change title** from "TrueSPA" to "True Single Page App behaviour"

**Change content as follows:**

"The WaveMaker app UI now behaves closer to a Single Page App functionality. This means, when user navigates between pages, only the content area is rendered again and the static area like header, leftnav, fotter, etc remain untouched. This provides a more pleasant user experience during page transitions.

Since this feature is still in beta phase, it is not enabled by default. Refer this doc[LINK_TO_HOWTO] on how to enable this behaviour in your app."